packages/client/mutations/handlers/handleGitHubCreateIssue.ts

```diff
@@ -0,0 +1,44 @@
+import {ConnectionHandler, RecordProxy, RecordSourceSelectorProxy} from 'relay-runtime'
+import SearchQueryId from '../../shared/gqlIds/SearchQueryId'
+import toTeamMemberId from '../../utils/relay/toTeamMemberId'
+import {CreateTaskMutationResponse} from '../../__generated__/CreateTaskMutation.graphql'
+import getGitHubIssuesConn from '../connections/getGitHubIssuesConn'
+
+const handleGitHubCreateIssue = (
```

Comment:
this function is so gross & i want a better way to do this.
the problem is I don't know a better way that also works with subscriptions.